app/helpers/hosts_helper.rb

```diff
@@ -315,7 +315,10 @@ def host_title_actions(host)
         ),
         button_group(
             link_to_if_authorized(_("Delete"), hash_for_host_path(:id => host).merge(:auth_object => host, :permission => 'destroy_hosts'),
-                                  :class => "btn btn-danger", :id => "delete-button", :data => { :message => _("Are you sure?") }, :method => :delete)
+                                  :class => "btn btn-danger",
+                                  :id => "delete-button",
+                                  :data => { :message => _("This action is not reversible. Are you sure you want to delete host #{host.name}?") },
```

Comment:
Use the "parameter" style [listed here](http://projects.theforeman.org/projects/foreman/wiki/Translating#Translating-for-developers) with `%s`, instead of `#{..}` inside a translated string.